cht/Nintendo - Nintendo Entertainment System/Legend of Zelda, The (USA, Europe) (Action Replay).cht

```diff
@@ -1,4 +1,4 @@
-cheats = 2 
+cheats = 37 
```

Comment:
Believe this should be 36?

Reply:
There are in fact 37 entries, but one of them has no code. That's because it's intended as a note - the Merchant Sells cheats can't be used together. It sounds like the format doesn't support notes, though. Super-easy to fix.